cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/EnergyConsumerConversion.java

```diff
@@ -42,4 +45,25 @@ private double p0() {
     private double q0() {
         return powerFlow().defined() ? powerFlow().q() : p.asDouble("qFixed", Double.NaN);
     }
+
+    private static void setLoadDetail(String type, Load load) {
+        if (type.endsWith("#ConformLoad")) { // can participate in scaling
```

Comment:
Indeed, what is important is that this kind of loads have a daily profile. Maybe you change comment ````ConformLoad represent loads that follow a daily load change pattern where the pattern can be used to scale the load with a system load.````